packages/babel-helper-member-expression-to-functions/src/index.ts

```diff
@@ -26,15 +27,17 @@ class AssignmentMemoiser {
     return value;
   }
 
-  set(key, value, count) {
+  set(key: t.Expression, value: t.LVal, count: number) {
     return this._map.set(key, { count, value });
   }
 }
 
-function toNonOptional(path, base) {
-  const { node } = path;
+function toNonOptional(
+  path: NodePath<t.Expression>,
+  base: t.Expression,
+): t.Expression {
   if (path.isOptionalMemberExpression()) {
```

Comment:
Nit: Can also use `t.isOptionalMemberExpression` so we can avoid multiple `node` access.